Bottom: 0f6b0b90d7739c61acae47649b0c406540570e73
Top:    10928aa10b4d15e0de5bc74c2afd50695e2ec668
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-16 01:50:43 +0100

Change scroll-wheel bindings to match more GTK applications









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 9b6e612..3b0b0d4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
